sbt_common/internal_model/src/main/scala/uk/ac/wellcome/models/work/internal/AbstractRootConcept.scala

```diff
@@ -1,28 +1,65 @@
 package uk.ac.wellcome.models.work.internal
 
+import uk.ac.wellcome.models.work.text.TextNormalisation._
+
 sealed trait AbstractRootConcept {
   val label: String
 }
 
 sealed trait AbstractConcept extends AbstractRootConcept
 
 case class Concept(label: String) extends AbstractConcept
+object Concept {
+  def normalised(label: String): Concept = {
+    Concept(trimTrailing(label, '.'))
+  }
+}
 
 case class Period(label: String) extends AbstractConcept
+object Period {
+  def normalised(label: String): Period = {
+    Period(trimTrailing(label, '.'))
+  }
+}
 
 case class Place(label: String) extends AbstractConcept
+object Place {
+  def normalised(label: String): Place = {
+    Place(trimTrailing(label, ':'))
```

Comment:
Is there a spec or ticket that explains what normalisations we're meant to be doing? A comment linking to that would be helpful.